ci/generate-npm-package.js

```diff
@@ -27,7 +27,7 @@ async function copyAssets() {
         await fs.ensureDir(folder);
 
         try {
-            const files = (await fs.readdir(builtPath)).filter((path) => !path.includes('patternlib'));
+            const files = (await fs.readdir(builtPath)).filter((path) => !path.includes('ds-img'));
```

Comment:
```suggestion
            const files = (await fs.readdir(builtPath)).filter((path) => !path.includes('ds-'));
```
This is what I was suggesting, it allows us to still have folders that start with this if we need to bundle other folders in there at some point which is what the original "patternlib" version did rather than specifically fully hardcoding one folder name